tests/checks/default/BUILD.bazel

```diff
@@ -0,0 +1,51 @@
+load("@io_bazel_rules_go//go:def.bzl", "go_library", "go_test", "go_binary")
+load("@io_bazel_rules_go//tests:bazel_tests.bzl", "bazel_test")
+
+default_checker_exists = """
+# TODO(samueltan): parameterize architecture-specific component of this filepath.
+checker="bazel-bin/external/io_bazel_rules_go/linux_amd64_stripped/default_checker"
+if [ -x "${checker}" ]; then
+  result=0
+else
+  echo "error: checker binary ${checker} does not exist" >&2
+  result=1
+fi
+"""
+
+bazel_test(
```

Comment:
I'm not sure checking whether the checker binary exists is meaningful. The `bazel_test`s are all built in the same workspace, so you may see the binary there from an earlier build.

A better way to verify this would be to check some code that would fail a default check, then make sure the failure really happens during a build.

Avoid using multiple `bazel_test` rules if possible by the way; they are very slow.

Reply:
The only default check we will have will be vet, which I plan to add later. For now, no checks will be run by default. I could have the checker print some kind of success message, but that doesn't seem very Go-ish. Do you have any other ideas for basic test cases to temporarily anchor the development of this framework?

Reply:
Maybe remove or disable this test for now and add a TODO to add it back after support for vet is added. This test should verify a failure is detected by the default analysis without needing to configure the check. Since nothing gets run by default yet, I don't think we can test it.

I think I mentioned elsewhere (but can't find the comment now) that I'd like to merge this onto a feature branch until it's more fleshed out, so it doesn't need complete test coverage yet.

Reply:
Ok, I have condensed this into a single test target that checks for a build failure and vet error message. I have commented this target out for now.

Merging this onto a feature branch sounds good to me. Is this something I need to create, or can you merge this change into the feature branch for me?